### PR TITLE
Sana Ullah: [Response] Fixed return responses in All endpoints

### PR DESCRIPTION
**Pull request by Sana-Ullah786**
This PR fixes the response of each **Endpoint**.
**Also, each test case is modified accordingly to handle new response**
**Ticket Link:**
https://app.asana.com/0/1204411152293714/1204601040566441/f

### CODE REVIEW

**Review by mtalhatahir2001 — APPROVED** [review]
kudos to you, LGTM 🚀

File: src/endpoints/auth/post/login.py [hunk 1]
```diff
@@ -6,18 +6,22 @@
...
+from src.responses import custom_response
 from src.schemas.user import UserSchemaToken
 
 
-@router.post("/token", status_code=status.HTTP_200_OK, response_model=UserSchemaToken)
+@router.post("/token", status_code=status.HTTP_200_OK, response_model=None)
 async def login_for_access_token(
     form_data: OAuth2PasswordRequestForm = Depends(), db: Session = Depends(get_db)
-) -> UserSchemaToken:
+) -> None:
```
Comment (mustansirMuz) [inline src/endpoints/auth/post/login.py:16]: The return type should be dict here

**Review by mustansirMuz — APPROVED** [review]
LGTM, good work!!

File: src/endpoints/auth/post/login.py (hunk 1 above)
Comment (Sana-Ullah786) [reply src/endpoints/auth/post/login.py:16]: Fixed

**Pull request merged**